src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceObject = {};
+
+  sourceString.split(';').forEach((source) => {
+    const trimmedSource = source.trim();
+
+    if (trimmedSource) {
+      const [key, value] = trimmedSource.split(':').map((item) => item.trim());
+
+      sourceObject[key] = value;
+    }
+  });
```

Comment:
[CODE STYLE] - For better readability, you can chain the methods on separate lines. This will make your code cleaner and easier to understand. Here is an example: 

javascript
sourceString
  .split(';')
  .forEach((source) => {
    const trimmedSource = source.trim();
    if (trimmedSource) {
      const [key, value] = trimmedSource.split(':').map((item) => item.trim());
      sourceObject[key] = value;
    }
  });
```